src/main/java/org/gitlab4j/api/Constants.java

```diff
@@ -199,6 +199,28 @@ public String toString() {
         }
     }
 
+    /** Enum to use for ordering the results of getProjects() of GroupsApi. */
+    public enum GroupProjectOrderBy {
+

```

Comment:
There is no need for this enum, the same enums are defined in ProjectOrderBy.

Reply:
Oh, thanks. I will remove it.